### ISSUE TITLE
Kodi Omega crashes on webdav

### ISSUE THREAD

**Issue by tgotowik** [xbmc/xbmc#25160]
### Describe the bug
On Kodi v21.0.0 with a remote source based on webdav (HTTPS) will crash after starting any kind of video.

## Expected Behavior
Going into the remote directory clicking on a video it will play the video.

## Actual Behavior
Going into the remote directory clicking on a video will load a time until the error the error message pops up: "One or more items failed to play. Check the log for more information about this message."
Trying the reload the directory afterwards does not work and says "Couldn't connect to network server".

After a restart of kodi, you can connect to the server until you try to play any kind of video.

## Possible Fix
N/A

### To Reproduce
1.  Add a webdav (HTTPS) source
2. Enter the movie section of the remote source
3. Click on any video

### Debuglog
The debuglog can be found here: https://paste.kodi.tv/uhatorudir.kodi

### Your Environment
Used Operating system:
 - [ x] Android
 - [x ] Windows

 - Operating system version/name: Windows 11, Android TV
 - Kodi version: v21.0.0

**Comment by neo1973** [comment]
Can't reproduce with the Flatpak nor with a Linux build of current master.

**Comment by ovladuk** [comment]
> ## Bug report
> ### Describe the bug
> On Kodi v21.0.0 with a remote source based on webdav (HTTPS) will crash after starting any kind of video.
> 
> ## Expected Behavior
> Going into the remote directory clicking on a video it will play the video.
> 
> ## Actual Behavior
> Going into the remote directory clicking on a video will load a time until the error the error message pops up: "One or more items failed to play. Check the log for more information about this message." Trying the reload the directory afterwards does not work and says "Couldn't connect to network server".
> 
> After a restart of kodi, you can connect to the server until you try to play any kind of video.
> 
> ## Possible Fix
> N/A
> 
> ### To Reproduce
> 1. Add a webdav (HTTPS) source
> 2. Enter the movie section of the remote source
> 3. Click on any video
> 
> ### Debuglog
> The debuglog can be found here: https://paste.kodi.tv/uhatorudir.kodi
> 
> ### Your Environment
> Used Operating system:
> 
> * [ x] Android
> * [x ] Windows
> * Operating system version/name: Windows 11, Android TV
> * Kodi version: v21.0.0

did you figure this out? im having the same problem

**Comment by Whomeyes** [comment]
Try disable http2 in advanced settings.xml

**Comment by tgotowik** [comment]
> Try disable http2 in advanced settings.xml

this did not help :(

**Comment by tgotowik** [comment]
> > ## Bug report
> > ### Describe the bug
> > On Kodi v21.0.0 with a remote source based on webdav (HTTPS) will crash after starting any kind of video.
> > ## Expected Behavior
> > Going into the remote directory clicking on a video it will play the video.
> > ## Actual Behavior
> > Going into the remote directory clicking on a video will load a time until the error the error message pops up: "One or more items failed to play. Check the log for more information about this message." Trying the reload the directory afterwards does not work and says "Couldn't connect to network server".
> > After a restart of kodi, you can connect to the server until you try to play any kind of video.
> > ## Possible Fix
> > N/A
> > ### To Reproduce
> > 
> > 1. Add a webdav (HTTPS) source
> > 2. Enter the movie section of the remote source
> > 3. Click on any video
> > 
> > ### Debuglog
> > The debuglog can be found here: https://paste.kodi.tv/uhatorudir.kodi
> > ### Your Environment
> > Used Operating system:
> > 
> > * [ x] Android
> > * [x ] Windows
> > * Operating system version/name: Windows 11, Android TV
> > * Kodi version: v21.0.0
> 
> did you figure this out? im having the same problem

Nope, i just went back to 20.4. This works like a charm. I also tried 22.0, but I have the same problems with that version

**Comment by tgotowik** [comment]
> Can't reproduce with the Flatpak nor with a Linux build of current master.

Current master is 20.5 ? Like I said all version pre 21.0 worked like a charm. All above 21.0 have the same issue.

**Comment by neo1973** [comment]
> Current master is 20.5 ? Like I said all version pre 21.0 worked like a charm. All above 21.0 have the same issue.

master is 22.0-ALPHA1 (with alpha 1 not being tagged), it's the latest development version. The Flatpak is on 21.0.

**Comment by SnowMeteors** [comment]
me too
